curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou.md

```diff
@@ -112,12 +112,7 @@ assert.deepEqual(
 
 ```js
 function whatIsInAName(collection, source) {
```

Comment:
```suggestion
function whatIsInAName(collection, source) {

```
I think that usually we leave a blank line here

Reply:
Looks like the blank line is on 115 - the diff on this one just isn't great.

Reply:
Oh, bad GitHub